server/autolinkclient/client.go

```diff
@@ -40,7 +40,7 @@ func NewClientPlugin(api PluginAPI) *Client {
 
 func (c *Client) Add(links ...autolink.Autolink) error {
 	for _, link := range links {
-		linkBytes, err := json.Marshal(&link)
+		linkBytes, err := json.Marshal(link)
```

Comment:
Good catch

Reply:
It's a lint-only issue, I think - same JSON, but was an annoyance.